### PR TITLE
Resolved  precious requests and Logo updated

### PR DESCRIPTION
**Pull request by Chandan-SM**

### CODE REVIEW

**Review by PatelNeelMahesh — APPROVED** [review]
You can merge pull request.

File: backend/package.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "backend",
```
Comment (PatelNeelMahesh) [inline backend/package.json]: Instead of backend write score_keeper / score_keeper_server

File: backend/server.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+app.use(cors());
+app.use(express.json());
+
+mongoose.set("strictQuery", false);
+
+// Set up a database connection using Mongoose
+mongoose
+  .connect(
+    "mongodb+srv://chandangowda:Gowda$123@score-keeper.nuzfevx.mongodb.net/?retryWrites=true&w=majority"
```
Comment (PatelNeelMahesh) [inline backend/server.js]: Should not keep sensitive details public, should use environment variables.

**Review by PatelNeelMahesh — APPROVED** [review]
You can merge current pull request, once a pull request is approved and asked changed are done merge it and for new changes create new PR.

File: backend/package.json (hunk 1 above)
Comment (Chandan-SM) [reply backend/package.json]: Done

File: backend/server.js (hunk 2 above)
Comment (Chandan-SM) [reply backend/server.js]: Changes made

**Pull request merged**